app/google-services.json

```diff
@@ -0,0 +1,107 @@
+{
+  "project_info": {
+    "project_number": "768065119425",
+    "firebase_url": "https://commcare-a57e4.firebaseio.com",
+    "project_id": "commcare-a57e4",
+    "storage_bucket": "commcare-a57e4.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:768065119425:android:bf4b2396fb9805b2",
+        "android_client_info": {
+          "package_name": "org.commcare.dalvik"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "768065119425-hg10t0osvjqrt6pne3toaobabn5gobqs.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyBoBKDzl5DZxo-yp__96V3E8jnri17OyTE"
```

Comment:
IS it OK for this ID to be public? Looks like there are some configurations where this is OK and some where it is not, just want to confirm that this key doesn't allow improper access 
http://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public